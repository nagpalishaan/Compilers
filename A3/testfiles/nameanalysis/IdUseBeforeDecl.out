Error at line 2: symbol 'print' has not been declared before this use!
Error at line 2: symbol 'a' has not been declared before this use!
