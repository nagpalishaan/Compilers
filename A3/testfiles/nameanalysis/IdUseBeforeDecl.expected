Error at line 2: symbol 'a' has not been declared before this use!
